core/lib/config/src/configs/base_token_adjuster.rs

```diff
@@ -0,0 +1,25 @@
+use serde::Deserialize;
+
+/// By default the ratio persister will run every 30 seconds.
+pub const DEFAULT_INTERVAL_MS: u64 = 30_000;
+
+#[derive(Debug, Clone, PartialEq, Deserialize)]
+pub struct BaseTokenAdjusterConfig {
+    /// How often to spark a new cycle of the ratio persister to fetch external prices and persis ratios.
+    #[serde(default = "BaseTokenAdjusterConfig::default_interval")]
+    pub price_polling_interval_ms: u64,
+}
+
+impl Default for BaseTokenAdjusterConfig {
+    fn default() -> Self {
+        Self {
+            price_polling_interval_ms: Self::default_interval(),
+        }
+    }
+}
+
+impl BaseTokenAdjusterConfig {
+    pub fn default_interval() -> u64 {
+        DEFAULT_INTERVAL_MS
+    }
```

Comment:
Not sure I agree
- The default should only be used in this file. In fact I shouldn't have made it public (changing this)
- Otherwise, the actual pub field could be used to fetch the actual value, not the default
- Even creating a getter here that returns the `Duration` and removes the need to convert to `Duration` seems overkill to me as this field is only used in one place as of now

WDYT?

Reply:
Oh, I didn't mean `fn default_interval() -> Duration`, I meant `fn price_polling_interval(&self) -> Duration`. I still think it makes sense to add it, even for a single use.
No objections on publicity of methods.